pkgs/development/python-modules/i3ipc/default.nix

```diff
@@ -1,24 +1,33 @@
 { stdenv, buildPythonPackage, fetchFromGitHub
 , enum-compat
-, xorgserver, pytest, i3, python
+, xorgserver, pytest, pytest-xvfb, pytest-asyncio, i3, python, xlib, xdpyinfo
+, makeFontsConf, coreutils
 }:
 
 buildPythonPackage rec {
   pname = "i3ipc";
-  version = "1.6.0";
+  version = "2.0.1";
 
   src = fetchFromGitHub {
     owner  = "acrisci";
     repo   = "i3ipc-python";
     rev    = "v${version}";
-    sha256 = "0sb525wvwcnikjaqzha94xr97r1gjys30csmaj17swlxgyczxvq5";
+    sha256 = "10zpbiw1gcndn439g1vxcdkxllwp02qcmaal4w7hi2rzgaw1xkdk";
```

Comment:
```
hash mismatch in fixed-output derivation '/nix/store/fjy3h88svwcspsx72fg0hv8mjiphzk89-source':
  wanted: sha256:10zpbiw1gcndn439g1vxcdkxllwp02qcmaal4w7hi2rzgaw1xkdk
  got:    sha256:0cbjck147g03slavd9mi1126vk713p4ar6wxd754nnbk3w0wsgcl
```
```suggestion
    sha256 = "0cbjck147g03slavd9mi1126vk713p4ar6wxd754nnbk3w0wsgcl";
```

Reply:
Ah seems that accidentally 2.0.1 was used for the version instead of 2.1.1. This is the hash for the latter